### ISSUE TITLE
llvm-general-pure-3.4.5.3 can't compile its test suite 

### ISSUE THREAD

**Issue by peti** [bscarlet/llvm-general#143]
Citing from http://hydra.cryp.to/build/1042569/nixlog/1/raw:

~~~
reprocessing test suite 'test' for llvm-general-pure-3.4.5.3...

test/Test.hs:2:18:
    Could not find module ‘LLVM.General.Test.Tests’
    Use -v to see a list of the files searched for.
~~~~

The module appears to be missing from the release tarball.

**Comment by bscarlet** [comment]
Thanks. Should be fixed on the llvm-3.4 and llvm-3.3 branches.

**Comment by peti** [comment]
It would be great if you could leave this ticket open and close it only after a version that contains the fix has been released to Hackage.

**Comment by bscarlet** [comment]
Um. I did. The fix is in 3.3.14.4 and 3.4.5.4, which I uploaded to haskell before my comment in which I said it was fixed. Browsing hackage, I see the new line in the .cabal file for those versions. Do you?

**Comment by peti** [comment]
Ah, okay. I didn't realize that you released new versions; your message sounded like the fix was in Git but not released yet. Anyway, thank you for the quick fix.
